src/app/components/app-images/app-images.component.ts

```diff
@@ -136,6 +138,9 @@ export class AppImagesComponent implements OnInit, OnDestroy {
           }
         } else if (output && output.errorDetail) {
           this.zone.run(() => this.terminalInput = `<span style="color:rgb(255,85,85);">${output.errorDetail.message}</span>`);
```

Comment:
I know this is from previous commits but can you unwrap this message from `<span>` please?

Reply:
sure, but then the error description in terminal will be white instead red, is that ok?

Reply:
I think I encountered this a couple of times and it was not parsed as span but as the whole text as it is here. I think its better to be white than with html text inside terminal output.

Reply:
Yes you're probably right, I'll remove span...

Reply:
done

Reply:
thanks.